src/Service/ImageDescriptionServiceInterface.php

```
@@ -0,0 +1,21 @@
+ */
+interface ImageDescriptionServiceInterface
+{
+    /**
+     * Generates a textual description for the given image file.
+     *
+     * @param UploadedFile $imageFile The uploaded image file.
+     * @param string|null $prompt An optional prompt to guide the description generation.
+     * @return string The generated textual description of the image.
+     * @throws \Exception If description generation fails.
+     */
+    public functiongenerateDescriptionForImage(UploadedFile $imageFile, ?string $prompt = null): string;
```

PHPStan (2.1.15)

Syntax error, unexpected '(', expecting T_VARIABLE on line 20


(phpstan.parse)